client/__tests__/components/productResponse.test.tsx

```diff
@@ -11,25 +10,6 @@ const membersDataApiResponse = {
 	products: membersDataApiItems,
 };
 
-describe('testing mdapiResponseReader', () => {
-	it('mdapiResponseReader returns mdaResponse when MembersDataApiItems is passed in', () => {
-		const mdaResponse = membersDataApiItems;
-		expect(mdapiResponseReader(mdaResponse).products).toEqual(
-			membersDataApiItems,
-		);
-	});
-
-	it('mdapiResponseReader returns mdaResponse when MembersDataApiResponse is passed in', () => {
-		const mdaResponse = membersDataApiResponse;
-		expect(mdapiResponseReader(mdaResponse)).toEqual(
-			membersDataApiResponse,
-		);
-	});
-
-	it('throws validation error if MDAPI Response is in incorrect format', () => {
-		const mdaResponse = '';
-		expect(() => mdapiResponseReader(mdaResponse as any)).toThrow(
-			'MDAPI Response is of invalid format',
-		);
-	});
+it('mdapiResponseReader returns mdaResponse when MembersDataApiResponse is passed in', () => {
+	expect(membersDataApiResponse).toEqual(membersDataApiResponse);
```

Comment:
I don't see any reason to keep this test

Reply:
Deleted - although it's handy to keep in mind we don't have much code coverage for lower level functions